spec/services/github_service_spec.rb

```diff
@@ -78,35 +79,41 @@ def build
 
   context 'create pull requests for repository' do
     let!(:repo) { create(:repository) }
-    let!(:new_pull_request_data) {[
-      double(
-        id: 1,
-        title: 'Test',
-        number: 1,
-        state: 'open',
-        html_url: 'http://www.gihub.com/prueba',
-        created_at: '2017-12-12 09:17:52',
-        updated_at: '2017-12-12 09:17:52',
-        closed_at: '2017-12-12 09:17:52',
-        merged_at: '2017-12-12 09:17:52'
-      )
-    ]}
-
-    let!(:updated_pull_request_data) {[
-      double(
-        id: 1,
-        title: 'Test',
-        number: 1,
-        state: 'closed',
-        html_url: 'http://www.gihub.com/prueba',
-        created_at: '2017-12-12 09:17:52',
-        updated_at: '2017-12-14 09:17:52',
-        closed_at: '2017-12-14 09:17:52',
-        merged_at: '2017-12-14 09:17:52'
-      )
-    ]}
+    let!(:new_pull_request_data) {
```

Comment:
Avoid using {...} for multi-line blocks.

Reply:
Cambia el bloque `{...}` por un `do ... end`